bravado/http_client.py

```diff
@@ -197,6 +197,9 @@ def __init__(self, session, request):
         self.session = session
         self.request = request
 
+    def result(self, *args, **kwargs):
```

Comment:
+1.  I'd like to divorce any specific HttpClient implementation from `bravado.mapping` in preparation for splitting out to a separate python library (more details to come).   Fido and Requests specifics should not be in `bravado.mapping`.